rotkehlchen/data/importer.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, db: DBHandler) -> None:
         self.db = db
         self.db_ledger = DBLedgerActions(self.db, self.db.msg_aggregator)
 
-    def _consume_cointracking_entry(self, csv_row: Dict[str, Any]) -> None:
+    def _consume_cointracking_entry(self, csv_row: Dict[str, Any], **kwargs: Any) -> None:
```

Comment:
too bored to add more args? I can let it pass like this for now, but only use `**kwargs` if (1) there is a dynamic amount of args. So a dict with too many different possible values to consider or (2) too many args -- still debatable.

Doing it like this now, gives us no benefit and makes us lose the typing checks. Mypy can't type **kwargs. Especially if you declare it as Any.